L1Trigger/Phase2L1ParticleFlow/src/PFAlgoBase.cc

```diff
@@ -150,31 +150,64 @@ void PFAlgoBase::doVertexing(std::vector<Region> &rs, VertexAlgo algo, float &pv
             h_dz->Fill( p.floatDZ(), std::min(p.floatPt(), 50.f) );
         }
     }
-    switch(algo) {
-        case OldVtxAlgo: {
-                             int imaxbin = h_dz->GetMaximumBin();
-                             pvdz = h_dz->GetXaxis()->GetBinCenter(imaxbin);
-                         }; break;
-        case TPVtxAlgo: {
-                            float max = 0; int bmax = -1;
-                            for (int b = 1; b <= lNBins; ++b) {
-                                float sum3 = h_dz->GetBinContent(b) + h_dz->GetBinContent(b+1) + h_dz->GetBinContent(b-1);
-                                if (bmax == -1 || sum3 > max) { max = sum3; bmax = b; }
-                            }
-                            pvdz = h_dz->GetXaxis()->GetBinCenter(bmax); 
-                        }; break;
-    }
+    // switch(algo) {
```

Comment:
you can also remove the histogramming part, as it's no longer used if the vertex is read externally.